plugins/reporters/trustsource/src/main/kotlin/TrustSourceModel.kt

```diff
@@ -22,34 +22,53 @@ package org.ossreviewtoolkit.plugins.reporters.trustsource
 import kotlinx.serialization.Serializable
 
 @Serializable
-data class TrustSourceModule(
+data class NewScan(
```

Comment:
Commit message: Please use a title that adheres to [Conventional Commits](https://www.conventionalcommits.org/).

Reply:
Got it !